yarn.lock

```diff
@@ -21406,6 +21409,7 @@ fsevents@^1.2.7:
     bluebird: 3.5.1
     joi: 13.4.0
     request: 2.87.0
+  checksum: 03f255cd61a90c15719972d41d8d2020309a11ea0de7b2c4df133d9af4e25c05f885d32a075f06ea3eed00dad06bd2cd115be91d9c234eb170c8857a5081d162
```

Comment:
You'll want to rebase this against main; these checksum changes should be fixed.